packages/flutter/lib/src/rendering/selection.dart

```diff
@@ -146,6 +171,14 @@ mixin Selectable implements SelectionHandler {
   /// in local coordinates.
   List<Rect> get boundingBoxes;
 
+  /// The [TextSelection] of this [Selectable].
+  @override
+  TextSelection? get textSelection;
```

Comment:
Does `Selectable` need to expose this, or can we keep this in `SelectedContent`?

Reply:
Selectable needs to specifically expose this so that we can get this information at the Selectable's level. When we're building our 'global' selection in routines that use Selectable(s), we know their individual selection

Reply:
Following up on this comment, it seems to me that this can be contained within `SelectedContent` or am I missing something?